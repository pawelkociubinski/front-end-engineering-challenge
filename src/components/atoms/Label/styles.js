import styled from 'styled-components';

export default {
  default: {
    color: '#DEDFE0',
  },
  active: {
    color: '#E30015',
  },
};

export const StyledLabel = styled.label`
  color: ${({ theme }) => theme.color};
  font-size: 16px;
`;
