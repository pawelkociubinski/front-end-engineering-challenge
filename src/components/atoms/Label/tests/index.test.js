import React from 'react';
import renderer from 'react-test-renderer';
import Label from '../index';

describe('<Label />', () => {
  it('renders correctly', () => {
    const component = renderer.create(<Label theme="default">Label</Label>).toJSON();
    expect(component).toMatchSnapshot();
  });
});
