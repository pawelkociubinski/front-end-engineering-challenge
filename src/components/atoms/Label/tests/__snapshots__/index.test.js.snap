// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Label /> renders correctly 1`] = `
.c0 {
  color: #DEDFE0;
  font-size: 16px;
}

<label
  className="c0"
>
  Label
</label>
`;
